### PR TITLE
Fixing old 404 pages [Removed]

### PR DESCRIPTION
**Pull request by radioblahaj**

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ 
+  <link rel="icon" type="image/jpeg" href="quarklogo.jpg" />
+  
+  <style>
+   h1 {
+   font-family: 'Roboto', sans-serif;
+   
+   a {
+   border-color:blue;
+   <style>
```
Comment (Sheshank-s) [inline 404.html]: Still wrong, you need to close the style tag and the a element

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <meta name="description" content="The Quark Editor: HTML CSS JS">
+  <meta name="author" content="Sheshank Shankar">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+ 
+  <link rel="icon" type="image/jpeg" href="quarklogo.jpg" />
+  
+  <style>
+   h1 {
+   font-family: 'Roboto', sans-serif;
+
```
Comment (Sheshank-s) [inline 404.html]: End h1 as well

File: 404.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   h1 {
+   font-family: 'Roboto', sans-serif;
+   
+   a {
+   border-color:blue;
+   <style>
+</head>
+<body>
+
+<center>
```
Comment (Sheshank-s) [inline 404.html]: You need to close <center>

**Review by Sheshank-s — CHANGES_REQUESTED** [review]

**Comment by Sheshank-s** [comment]
Please fix the rest of the comments i gave

**Comment by Narulacodes** [comment]
i did

File: 404.html (hunk 3 above)
Comment (radioblahaj) [reply 404.html]: close what? Body?

**Comment by MasterOfTheTiger** [comment]
Finally it gets merged!

**Comment by Sheshank-s** [comment]
Yeah lol

**Pull request merged**